lib/active_campaign/client.rb

```diff
@@ -61,6 +61,7 @@ def create_request(method, api_method, options = {})
 
         request.headers              = { "User-Agent" => user_agent }
         request.auth.ssl.verify_mode = :none
+        request.auth.ssl.ssl_version = :TLSv1
```

Comment:
Will this work for everyone? For us it didn't or it was sslv3 that didn't work. Never the less I have a feeling we should make this configurable and also add some information about it in the readme?